pkg/etcdcli/etcdcli.go

```diff
@@ -49,27 +51,24 @@ type etcdClientGetter struct {
 
 func NewEtcdClient(kubeInformers v1helpers.KubeInformersForNamespaces, networkInformer configv1informers.NetworkInformer, eventRecorder events.Recorder) EtcdClient {
 	return &etcdClientGetter{
-		nodeLister:            kubeInformers.InformersFor("").Core().V1().Nodes().Lister(),
-		endpointsLister:       kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().Endpoints().Lister(),
-		networkLister:         networkInformer.Lister(),
-		nodeListerSynced:      kubeInformers.InformersFor("").Core().V1().Nodes().Informer().HasSynced,
-		endpointsListerSynced: kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().Endpoints().Informer().HasSynced,
-		networkListerSynced:   networkInformer.Informer().HasSynced,
-		eventRecorder:         eventRecorder.WithComponentSuffix("etcd-client"),
+		nodeLister:             kubeInformers.InformersFor("").Core().V1().Nodes().Lister(),
+		configmapsLister:       kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().ConfigMaps().Lister(),
+		networkLister:          networkInformer.Lister(),
+		nodeListerSynced:       kubeInformers.InformersFor("").Core().V1().Nodes().Informer().HasSynced,
+		endpointsListerSynced:  kubeInformers.InformersFor(operatorclient.TargetNamespace).Core().V1().Endpoints().Informer().HasSynced,
```

Comment:
this no longer needs the endpoint lister because it will simply blip on upgrades and the new contorl loop will fill in the configmap

Reply:
Removed